lesson8-UnitTesting/FizzBuzz/tests/FizzBuzzTests/Tests.cs

```diff
@@ -0,0 +1,35 @@
+using FizzBuzz;
+using NUnit.Framework;
+using System;
+
+namespace FizzBuzzTests
+{
+    public class Tests
+    {
+        [Test]
+        [TestCase(1, "1")]
+        [TestCase(3, "Fizz")]
+        [TestCase(4, "4")]
+        [TestCase(5, "Buzz")]
+        [TestCase(15, "FizzBuzz")]
+        [TestCase(90, "FizzBuzz")]
+        [TestCase(95, "Buzz")]
+        [TestCase(99, "Fizz")]
+        [TestCase(100, "Buzz")]
+        public void Calc(int input, string expectedResult)
```

Comment:
Test names should follow the pattern `Action_ExpectedResult_Condition`. E.g. in this case `Generate_ShouldReturnExpected_ForNumberWithinRange`.

Reply:
done